modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/upgrade/v1_0_1/DLFolderNameUpgradeProcess.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.image.uploader.web.internal.upgrade.v1_0_1;
+
+import com.liferay.document.library.kernel.model.DLFolder;
+import com.liferay.document.library.kernel.service.DLFolderLocalService;
+import com.liferay.image.uploader.web.internal.util.UploadImageUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Lily Chi
+ */
+public class DLFolderNameUpgradeProcess extends UpgradeProcess {
+
+	public DLFolderNameUpgradeProcess(
+		DLFolderLocalService dlFolderLocalService) {
+
+		_dlFolderLocalService = dlFolderLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		long folderId = 0;
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				"select folderId from DLFolder where name = ?")) {
+
+			preparedStatement.setString(1, "java.lang.Class");
+
+			ResultSet resultSet = preparedStatement.executeQuery();
+
+			while (resultSet.next()) {
+				folderId = resultSet.getLong("folderId");
+			}
+		}
+
+		DLFolder dlFolder = _dlFolderLocalService.getDLFolder(folderId);
+
+		dlFolder.setName(UploadImageUtil.TEMP_IMAGE_FOLDER_NAME);
+
+		_dlFolderLocalService.updateDLFolder(dlFolder);
+	}
+
```

Comment:
@slnn, I find there are two issues in the fix.

1. From code logic seen, you should move the blow codes to while loop.  You may directly use `_dlFolderLocalService.getDLFolder(resultSet.getLong("folderId"))` to get the Folder. This also will avoid executing folderId is 0 possibility.
```
DLFolder dlFolder = _dlFolderLocalService.getDLFolder(folderId);

		dlFolder.setName(UploadImageUtil.TEMP_IMAGE_FOLDER_NAME);

		_dlFolderLocalService.updateDLFolder(dlFolder);
```
2. Directly use the sql will get unexpected result. For example,  the user created one folder "java.lang.Class" in the UI. The fix will change it and this is not expected behavior. Please try to get specific dlFolder and update it. For example, I know we want to update the dlfolder which was from user profile. Please try to get the folder by using user profile info to get the folder.